spacy_llm/tasks/templates/span-srl.v1.jinja

```diff
@@ -0,0 +1,25 @@
+You are an expert Semantic Role Labeling (SRL) system. Your task is to accept Text as input and extract the Predicates and the Semantic Roles for each Predicate's ARGs in a step-by-step manner.
+{# whitespace #}
+{%- if predicates -%}
+Step 1: Use the following Predicates for the Text:
+Predicates: {{predicates}}
+{%- else -%}
+Step 1: Extract the Predicates for the Text in the following format :
+Predicates: <comma delimited list of strings>
+{%- endif -%}
+{# whitespace #}
+Step 2: For each Predicate, extract only the following Sematic Roles in '''Text''' in this format:
+Text: <string corresponding to the provided Text>
+Predicate: <string corresponding to the current Predicate>
+ARG-0 (Agent): <string corresponding to the Agent>
+ARG-1 (Patient or Theme): <string corresponding to the Patient or Theme>
+ARG-2: <string corresponding to the ARG-2>
+ARG-M-LOC (Location Modifier): <string corresponding to the Location Modifier>
+ARG-M-TMP (Temporal Modifier): <string corresponding to the Temporal Modifier>
```

Comment:
Is it possible users would want to use different arguments?

Reply:
good point. I have completely ignored adding label definitions rendering in this prompt. I am working on v2 of this prompt, which has this

Reply:
You don't need to bump the version, as the task hasn't been in any released versions yet. Just modify `v1`.

Reply:
done

Reply:
https://github.com/explosion/spacy-llm/pull/226/commits/6ab47237b694b5721e7290e7f01ee5ed7b77e6a9